zizmor/action.yaml

```diff
@@ -0,0 +1,87 @@
+# Copyright 2025 Chainguard, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+name: "GitHub Actions SCA with zizmor"
+description: |
+  This action scans a repository's Workflow footprint with zizmor.
+
+inputs:
+  octo_sts_token:
+    description: "octo-sts token for online scans; if empty, scans will be run as 'offline'"
+    required: false
+    default: ""
+  persona:
+    description: "Persona for zizmor scans (auditor, regular, pedantic)"
+    required: false
+    default: "regular"
+  scan_category:
+    description: "Category to store scan results under"
+    required: false
+    default: "zizmor"
+  upload_results:
+    description: "Whether to upload .sarif scan results; if false, output format will be 'github'"
+    required: false
+    default: "false"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install Rust
+      uses: dtolnay/rust-toolchain@fcf085fcb4b4b8f63f96906cd713eb52181b5ea4 # stable
+      with:
+        toolchain: stable
+
+    - name: Install zizmor
+      shell: bash
+      run: |
+        cargo install --locked zizmor
```

Comment:
why not make the action to use the image `ghcr.io/woodruffw/zizmor:1.5.2` instead of downloading? just a thought

Reply:
I was thinking that as well. We have a Wolfi package for this too.

Reply:
would use the package if that is public :)

Reply:
Updated in [`c8a5162` (#530)](https://github.com/chainguard-dev/actions/pull/530/commits/c8a5162c7de1fda929ad70bcaeacdb65f64daa66).